languages/swift/config.json

```diff
@@ -30,6 +30,12 @@
         ],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "enums",
+        "uuid": "0a634d93-d073-4a2b-b6c4-1ba53c04b2bb",
+        "concepts": ["enumerations"],
+        "prerequisites": ["string-components", "functions"]
```

Comment:
I don't think there is a `functions` concept unlocked as a concept.

Reply:
Huh. I somehow left `functions` out of the `basics` concept exercise concepts (along with a couple of other concepts). Fixing that in this next commit.